definitions/ext-snowflake_account/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: EXT
+type: SNOWFLAKEACC
```

Comment:
The folder name needs to be `domain-type` so in your case it'd be `ext-snowflakeacc`